src/briefcase/commands/base.py

```diff
@@ -55,6 +54,50 @@ def __init__(self, platform):
         )
 
 
+class Log:
```

Comment:
As a structural matter, I'd argue that this should be part of the new console module as well. Although it will only be used by Commands in practice, there's nothing Command-specific about what it's doing.

Reply:
makes sense. moved to `console.py`.